app/test_main.py

```diff
@@ -1,3 +1,48 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected",
+    [
+        (0, 0, [0, 0]),
+        (-2, -3, [0, 0]),
+        (-2, 3, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (100, 100, [21, 17])
+    ],
+    ids=[
+        "both values are zero",
```

Comment:
Use more informative descriptions here: e.g. `0 cat and dog age must equal 0 in human age`